doc/upgrading.md

```diff
@@ -72,6 +75,10 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 
 ## Version 25
 
+* PostgresClient constructor now, by default, starts Embedded Postgres.
+  This means that if you have unit tests that do not start Embedded Postgres
+  explicitly, you'll still need to terminate Embedded Postgres by calling
+  `PostgresClient.stopEmbeddedPostgres`.
```

Comment:
https://github.com/folio-org/raml-module-builder/commit/dbfbde47477f39244889e9f99e0df6917f88f5ce#diff-6e1e3df55057077885afd83e33ea0f65R3413

That pull request was part of RMB 26, not 25.

That pull request automatically starts embedded postgres, but it also added `Runtime.getRuntime().addShutdownHook(new Thread(PostgresClient::stopEmbeddedPostgres));` to automatically stop embedded Postgres. Can you point me to an example where a manual `PostgresClient.stopEmbeddedPostgres` is needed?

Reply:
mod-email .. https://github.com/folio-org/mod-email/commit/19deea6c8c4c76d29cd148690f3643822fbdf084
and see https://issues.folio.org/browse/MODEMAIL-32